src/example-types/terms_stats.js

```diff
@@ -2,11 +2,25 @@ var _ = require('lodash/fp')
 var esTwoLevel = require('./esTwoLevelAggregation').result
 let { buildRegexQueryForWords } = require('../regex')
 let { getField } = require('../fields')
+let { metrics, hasValidMetrics } = require('../aggUtils')
 
 module.exports = {
   validContext: context => context.key_field && context.value_field,
   async result(context, search, schema) {
     let field = getField(schema, context.key_field, context.fieldMode)
+    let orderPaths = _.reduce(
+      (obj, metric) =>
+        _.extend(
+          {
+            [metric]: {
+              [`twoLevelAgg_${metric}.value`]: context.sortDir || 'desc',
```

Comment:
It seems like this code path would produce the same results as the existing code path so nothing in this file needed to change